### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by hurricanerix**
Moved a lot of things around.

### CODE REVIEW

File: assets/shaders/dev.frag [hunk 1]
```diff
@@ -12,12 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#version 410 core
+#version 120
 
-out vec4 fragColor;
+// out vec4 FragColor;
```
Comment (aerwin3) [inline assets/shaders/dev.frag]: If this needs to be removed go ahead and delete it instead of having cruft.

File: core/dev/dev.go [hunk 2]
```diff
@@ -128,35 +129,24 @@ func (ctx *Context) Update() {
...
 func createDevWindow() *glfw.Window {
 	var devWindow *glfw.Window
 	glfw.WindowHint(glfw.Resizable, glfw.True)
-	glfw.WindowHint(glfw.ContextVersionMajor, 4)
+	glfw.WindowHint(glfw.ContextVersionMajor, 2)
 	glfw.WindowHint(glfw.ContextVersionMinor, 1)
-	glfw.WindowHint(glfw.OpenGLProfile, glfw.OpenGLCoreProfile)
-	glfw.WindowHint(glfw.OpenGLForwardCompatible, glfw.True)
+	// glfw.WindowHint(glfw.OpenGLProfile, glfw.OpenGLCoreProfile)
+	// glfw.WindowHint(glfw.OpenGLForwardCompatible, glfw.True)
```
Comment (aerwin3) [inline core/dev/dev.go]: Do we need these or can they be removed?

File: assets/shaders/dev.frag (hunk 1 above)
Comment (hurricanerix) [inline assets/shaders/dev.frag]: test

Comment (hurricanerix) [inline assets/shaders/dev.frag]: It could be removed, but will be needed when we move to newer GLSL versions.

File: core/dev/dev.go (hunk 2 above)
Comment (hurricanerix) [inline core/dev/dev.go]: They will be needed when we move to newer OpenGL versions. I just left them there so I don't have to look it up again.

**Comment by aerwin3** [comment]
LGTM after a discussion.

**Pull request merged**